views/apartments/edit.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Update Information for <%= @apartment.address %></h1>
+<form action="/apartments/<%= @id %>" method="post">
```

Comment:
I recommend `@apartment.id` since memory has already been allocated for `@apartment`